pkg/statistics/handle/types/interfaces.go

```diff
@@ -158,6 +158,12 @@ type StatsAnalyze interface {
 	// CheckAnalyzeVersion checks whether all the statistics versions of this table's columns and indexes are the same.
 	CheckAnalyzeVersion(tblInfo *model.TableInfo, physicalIDs []int64, version *int) bool
 
+	// OnBecomeOwner is used to handle the event when the current TiDB instance becomes the stats owner.
+	OnBecomeOwner()
```

Comment:
I think embedding the `Listener` interface is better

Reply:
Updated.